pyspecProcScripts/load_data.py

```diff
@@ -333,7 +333,8 @@ def proc_Hahn_echoph(s, fl=None):
     s *= s.shape["nScans"]
     s.squeeze()
     s.reorder(["ph2", "ph1"])
-    s.setaxis("nScans", r_[0:nScans])
+    if "nScans" in s.dimlabels:
+        s.setaxis("nScans", r_[0:nScans])
```

Comment:
proc_Hahn_echoph.py was complaining about this so I edited so it runs

Reply:
specifically the squeeze above gets rid of the nScans dimension if there is only 1 scan

Reply:
```
s.setaxis('nScans','#')
```